cloudflare.go

```diff
@@ -397,3 +397,18 @@ type WAFRulesResponse struct {
 		TotalCount uint `json:"total_count"`
 	} `json:"result_info"`
 }
+
+type PurgeZoneCacheRequest struct {
```

Comment:
Sounds good to me, I wasn't aware of the `omitempty` tag. I first tried to put both the file and site purge into the same request, but got an error back from the API that only one of `files` and `purge_everything` was allowed, even though I only had set one of them. This has also been made in my latest change.